rules/endpoint/elastic_endpoint.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint alert is received. Enabling this rule allows you to
+immediately begin investigating your Elastic Endpoint alerts.
+"""
+enabled = true
+from = "now-15m"
```

Comment:
What `from`/`interval` will we want for each of these? Also, I imagine we'll want to increase the `max_signals` as well?

Reply:
Let's change to `"now-10m"`, with interval `5m`